docs/_static/spec/server.yml

```diff
@@ -893,6 +927,18 @@ components:
             Tracker:
               $ref: '#/components/examples/Tracker'
 
+    200Story:
+      description: Success
+      content:
+        text/markdown:
+          example: >-
+            ## story_00055028
+            * greet: hello
+             - utter_ask_howcanhelp
+            * inform{"cuisine": "indian", "people": "two", "price": "moderate"}: I'm looking for a moderately priced Indian restaurant for two people
```

Comment:
isn't this a different format? `md_format_message()` should generate something like `*inform: I'm looking for a [moderately priced](price:moderate)...`